### PR TITLE
dao of log entities

### PR DESCRIPTION
**Pull request by charlliz**

### CODE REVIEW

File: src/main/java/cz/fi/muni/pa165/Model/Dao/RentRecordImpl.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import java.util.UUID;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+
+/**
+ *
+ * @author charlliz
+ */
+public class RentRecordImpl implements RentRecordDao{
```
Comment (jakubsarmir) [inline src/main/java/cz/fi/muni/pa165/Model/Dao/RentRecordImpl.java]: anotation for repository missing

**Comment by charlliz** [comment]
opravene

**Comment by jkuchar** [comment]
Huh! Who is going to write integration tests for this? Don't you think there should be also common interface for obtaining all records?

**Comment by charlliz** [comment]
I don't know who will write integration tests. You have free hands you can write common interface if you want.

**Pull request merged**